ui/src/app/components/gke-app-configuration-panels/create-gke-app.spec.tsx

```diff
@@ -4,24 +4,26 @@ import * as React from 'react';
 
 import {
   AppsApi,
+  AppStatus,
```

Comment:
the changes in this file are mainly cleanup/refactoring until L290 where I added tests for the new functionality